source/extensions/filters/http/local_ratelimit/local_ratelimit.h

```diff
@@ -163,7 +163,7 @@ using FilterConfigSharedPtr = std::shared_ptr<FilterConfig>;
  */
 class Filter : public Http::PassThroughFilter, Logger::Loggable<Logger::Id::filter> {
 public:
-  Filter(FilterConfigSharedPtr config) : config_(config) {}
+  Filter(FilterConfigSharedPtr config) : config_(config), used_config_(config_.get()) {}
```

Comment:
initialize the used_config_ with the default global config_. This ensure our used_config_ will always has valid value.